.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+### Describe the change
+
+Explanation of what this PR does
+
+### Steps to test the PR
+
+Recommendations for how to test this PR
```

Comment:
```suggestion
Recommendations for how to test this PR.  Reminder that each PR should also include a "Test" label.
```

Non-blocking suggestion.  We could add to our process that each PR must add one of the test labels.